docs/book/src/reference/providers.md

```diff
@@ -29,6 +29,7 @@ updated info about which API version they are supporting.
 - [Sidero](https://github.com/talos-systems/sidero)
 - [Tencent Cloud](https://github.com/TencentCloud/cluster-api-provider-tencent)
 - [vSphere](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere)
+- [KubeVirt](https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt)
```

Comment:
Providers are in alphabetical order. could you move Kubevirt up in the list?

Reply:
Thank you @fabriziopandini. I've made the required changes. Can you help to solve the recent failed test?